mlflow/pyfunc/model.py

```diff
@@ -83,6 +84,35 @@ class PythonModel:
 
     __metaclass__ = ABCMeta
 
+    def __new__(cls, *args, **kwargs):
+        cls._warn_on_setting_model_in_init()
+        return super().__new__(cls)
+
+    @classmethod
+    def _warn_on_setting_model_in_init(cls):
+        message = (
```

Comment:
got rid of the color, added code sample:
<img width="719" alt="Screenshot 2023-11-20 at 1 23 28 PM" src="https://github.com/mlflow/mlflow/assets/148037680/a928387f-c516-4f59-b077-d78791f573a6">